assets/src/application/ui/layout/espressoTable/EspressoDragSortingTable/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Table } from 'antd';
+import { DndProvider, DragSource, DropTarget } from 'react-dnd';
+import { pipe, update } from 'ramda';
+import HTML5Backend from 'react-dnd-html5-backend';
```

Comment:
This will be replaced with actual GQL mutation upon implementing this in real component.